frontend/src/components/Admin/TeamEvent/TeamEventForm.tsx

```diff
@@ -145,7 +147,23 @@ const TeamEventForm = (props: Props): JSX.Element => {
             onChange={() => setTeamEventHasHours(false)}
           />
         </Form.Field>
-
+        <label className={styles.label}>Is this a community event?</label>
+        <Form.Field>
+          <Radio
+            label="Yes"
+            value="Yes"
+            checked={isCommunity}
+            onChange={() => setIsCommunity(true)}
+          />
+        </Form.Field>
+        <Form.Field>
+          <Radio
+            label="No"
+            value="No"
+            checked={!isCommunity}
+            onChange={() => setIsCommunity(false)}
```

Comment:
<img width="261" alt="Screen Shot 2022-11-15 at 3 04 21 PM" src="https://user-images.githubusercontent.com/70241445/202014789-9e7e135f-2c57-4248-a49f-a0d1d992d95b.png">
Would it be better to have the 'yes' and 'no' radio buttons in line so that the form isn't unnecessarily long?

Reply:
@airaagrawal1 both are good points (radio button and additional categories), let's discuss with @JacksonStaniec during work session today?

Reply:
Discussed offline: 
The community category itself is an experimental feature that the leads might not opt to use next semester, or might not continue to use if the next semester doesnt go as well as planned. So, we shouldnt invest the development resources into making a system of adding additional categories until we are more certain it will be used. We think the probability of having additional category requirements in the future is relatively low anyways, based on the current difficulty for many members to achieve 3 uncategorized credits. For the radio button, we decided taking up less vertical space is better.